php/Dockerfile

```diff
@@ -53,9 +53,17 @@ RUN apt-get update && \
     ln -s /usr/lib/x86_64-linux-gnu/ImageMagick-6.8.9/bin-Q16/MagickWand-config /usr/bin/ && \
     rm -rf /var/lib/apt/lists/*
 
-# Install Apcu, Igbinary, Imagick, Redis and ssh2
-RUN pecl install apcu igbinary imagick redis ssh2-1.1.2 && \
-    docker-php-ext-enable apcu igbinary imagick redis ssh2
+# Install Apcu, Igbinary, Imagick, Redis
+RUN pecl install apcu igbinary imagick redis && \
+    docker-php-ext-enable apcu igbinary imagick redis
+
+# Clone, build and install SSH2 extension from source
+RUN \
+    cd /tmp && \
+    git clone https://git.php.net/repository/pecl/networking/ssh2.git && \
+    cd /tmp/ssh2/ && \
+    .travis/build.sh && \
```

Comment:
I don't know how the script is written, but perhaps you could execute the script from your current location to avoid the usage of `cd` instructions?

Reply:
I tried to do it, unfortunately I get an error when building directly with `/tmp/ssh2/.travis/build.sh` :/